c7n/cli.py

```diff
@@ -240,10 +240,15 @@ def setup_parser():
         "--skip-validation",
         action="store_true",
         help="Skips validation of policies (assumes you've run the validate command seperately).")
+
+    metrics_help = ("Emit metrics to provider metrics. Specify 'aws', 'gcp', or 'azure'. "
+        "To emit metrics back to the master account in aws: `-m aws://master`. To emit metrics "
```

Comment:
i kinda of think we should reference the provider specific docs here for more information and omit the additional verbage on aws vs a doc link. the notion of a master account is not obvious without additional context, and its not really appropriate to expand on it here as a provider specific reference.

Reply:
ie. this would be better to replace with `For additional aws specific options see https://doc_url`